harvester/metadata/tests/test_harvest_study_vocabulary.py

```diff
@@ -0,0 +1,40 @@
+from unittest.mock import patch
+from datetime import datetime
+
+from django.test import TestCase, override_settings
+from django.core.management import call_command, CommandError
+from django.contrib.auth.models import User
+from metadata.models import MetadataField, MetadataValue
+
+
+@override_settings(VERSION="0.0.1")
+class TestHarvestStudyVocabulary(TestCase):
+    """
+    This test case represents the scenario where a harvest is started from t=0
+    """
+
+    fixtures = ["initial-study-vocabulary-resources"]
+    spec_set = None
+    repository = None
+
+    def setUp(self):
+        super().setUp()
+        self.user = User.objects.create(username="supersurf")
+        self.client.force_login(self.user)
+
+    def test_no_duplicate_error(self):
```

Comment:
Tests fulfill two goals. One is testing that things work like expected. The other is documenting to others and ourselves how we expect things to work. This test fulfills the first criteria, but needs improvement on the second.

I've made you reviewer of these two PR's. I'll close them in a minute, but it might be interesting to look at them to see how the "message" argument to assert calls can be used to document what's going on.
* https://github.com/surfedushare/sources-middleware/pull/6
* https://github.com/surfedushare/sources-middleware/pull/7

You could add two asserts after each command that checks that the http://purl.edustandaard.nl/concept/27aee99f-1b5f-45ba-84e9-4a52c1d46a63 value exists exactly one time. With the second assert the message can be something like: "Expected a second call the harvest_study_vocabulary not to duplicate values". That then tells us a bit about how  harvest_study_vocabulary is implemented.